packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -923,4 +881,70 @@ export default class Editor {
     }
 
     //#endregion
+
+    //#region Dark mode APIs
+
+        /**
+     * Set the dark mode state and converts the content.
+     * @param desiredDarkMode The desired status of dark mode. True if the editor should be in dark mode, false if not.
+     */
+    public setDarkModeState(desiredDarkMode?: boolean) {
+        if (this.isDarkMode() != desiredDarkMode) {
+            const currentContent = this.getContent(true /* triggerExtractContentEvent */, true /* getSelectionMarker */);
+            this.core.inDarkMode = desiredDarkMode;
+            if (desiredDarkMode) {
+                this.setContent(currentContent, true /* triggetContentChangedEvent */, true /* convertToDarkMode */);

```

Comment:
ditto on line style

Reply:
typo: trigget

Reply:
We don't actually care about the value of triggerContentChanedEvent here, we just want to use the default.

In that case I think you can pass `undefined`

https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters
>In TypeScript, we can also set a value that a parameter will be assigned if the user does not provide one, or if the user passes undefined in its place. These are called default-initialized parameters. Let’s take the previous example and default the last name to "Smith".

Reply:
Oh, didn't know undefined had that behavior. Good call.